### PR TITLE
Fix/peers db penalize fix

### PR DESCRIPTION
**Pull request by i-skrypnyk**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
+2.0.0-RC7
+---------
+* CI/CD implementation
+* Delivery Tracker - limit requests map per peer
+* add multiple fields to the /connectedPeer endpoint response
```
Comment (i-Alex) [inline release-notes.md]: @paologalligit pls confirm, I think we don't have this, it is going to be added directly to the SDK, right?

Comment (paologalligit) [reply release-notes.md]: Correct, this is just an sdk change

Comment (i-skrypnyk) [reply release-notes.md]: so this one won't be merged now?
https://github.com/HorizenOfficial/Sparkz/pull/8

Comment (paologalligit) [reply release-notes.md]: No, we're going to close it once the sdk-97 on Sidechains-SDK will be merged

Comment (i-skrypnyk) [reply release-notes.md]: Ok, removed that line, thanks

File: src/test/scala/sparkz/core/network/peer/InMemoryPeerDatabaseSpec.scala [hunk 2]
```diff
@@ -192,11 +193,37 @@ class InMemoryPeerDatabaseSpec extends NetworkTests {
     }
   }
 
-  private def generatePeers(ammount: Int) = {
+  it should "not update penalty timestamp on consecutive violations during safeInterval" in {
```
Comment (i-Alex) [inline src/test/scala/sparkz/core/network/peer/InMemoryPeerDatabaseSpec.scala:196]: this test is failing for me.

**Review by i-Alex — CHANGES_REQUESTED** [review]
The test is failing for me.
We will include this fix into the next release.
Meanwhile I have decoupled the release notes from this one.

File: src/test/scala/sparkz/core/network/peer/InMemoryPeerDatabaseSpec.scala (hunk 2 above)
Comment (i-Alex) [reply src/test/scala/sparkz/core/network/peer/InMemoryPeerDatabaseSpec.scala:196]: is good now

File: src/main/resources/reference.conf [hunk 3]
```diff
@@ -190,7 +190,7 @@ sparkz {
 
     # Misbehaving peer penalty score will not be increased withing this time interval,
     # unless permanent penalty is applied
-    penaltySafeInterval = 5m
+    penaltySafeInterval = 1m
```
Comment (i-Alex) [inline src/main/resources/reference.conf:193]: why do we need to change the value from 5m to 1m?

**Review by i-Alex — APPROVED** [review]

**Comment by i-skrypnyk** [comment]
Closing in favor of https://github.com/HorizenOfficial/Sparkz/pull/16

**Pull request closed without merging**